wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/404.scss

```diff
@@ -0,0 +1,101 @@
+@import 'wagtailadmin/scss/variables';
+@import 'wagtailadmin/scss/mixins';
+@import 'wagtailadmin/scss/grid';
+@import 'wagtailadmin/scss/components/logo';
+
+body {
```

Comment:
This is all new code so I think it should be held to as high as possible standards. No selectors on bare elements please, use a class (using the `{% block bodyclass %}`?).